test/test_tensorexpr.py

```diff
@@ -1234,5 +1234,19 @@ def test(x):
         assert torch.allclose(scripted(a), 2 * a)
         assert cx.elapsed_value() == 1
 
+    def test_eig_argument(self):
```

Comment:
Why is this test in test_tensorexpr?

Reply:
This was my best guess for an existing file to put it in. Should it go in test_mkldnn.py or somewhere else?

Reply:
Take a look at test/test_torch.py, it has existing tests for torch.eig()